java-demo/Dockerfile

```
@@ -0,0 +1,38 @@
+FROM eclipse-temurin:21-jdk-alpine AS builder
+
+WORKDIR /build
+
+COPY ./.mvn ./.mvn
+COPY ./mvnw ./mvnw
+COPY ./pom.xml ./pom.xml
+COPY ./src ./src
+
+RUN ./mvnw clean package
+
+FROM eclipse-temurin:21-jre-alpine AS runtime
+
+# Common base image.
+LABEL org.opencontainers.image.authors="matcaz@noc.ac.uk"
+LABEL org.opencontainers.image.description="Example OCI image for a Python application."
```

This probably needs a small correction?
```suggestion
LABEL org.opencontainers.image.description="Example OCI image for a Java application."
```